### ISSUE TITLE
Simplify installation

### ISSUE THREAD

**Issue by ad-si** [purescript/purescript#3684]
I tried to install PureScript in a Docker container with a [GraalVM](https://www.graalvm.org) backend. But because of some hacky postinstall steps it fails. Check out https://github.com/graalvm/graaljs/issues/157#issuecomment-492129793 for details.

Can we simplify this?

**Comment by hdgarrood** [comment]
Yes we can! In fact I think this is already fixed as of `purescript-installer@0.2.0`, which I just published (which will take the place of `install-purescript-cli`). Can you give it a try with that please? We haven't yet published a new version of the `purescript` npm package so in the meantime you can run
```
$ npm install [--global] purescript-installer
$ install-purescript --purs-ver 0.13.0
```
Since this is an issue with https://github.com/purescript/npm-installer and we prefer to track those separately from the compiler, I'm going to close this, but please do open a new issue there if this is not resolved.

**Comment by ad-si** [comment]
Yes, works! 😁
Thanks for your work on it!

Btw: It should be possible to move the issue with a button on the right side. So if you think this issue should be in another repository, you can simply move it there.
